src/styles/index.scss

```diff
@@ -1,3 +1,273 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  --hover-color: var(--blue);
+
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  padding: 0 40px;
+  display: flex;
+  align-items: center;
+  border: 2px solid var(--blue);
+  justify-content: space-between;
+}
```

Comment:
The `.header` class has a border color defined with `var(--blue)`. According to the code style rules, it's recommended to use a variable for all colors, which you've already defined as `--blue`. However, ensure that the variable name is semantic and represents the usage or the color itself.